README.md

```diff
@@ -132,6 +132,8 @@ MSVC builds of Rust additionally require an installation of Visual Studio 2017
 (or later) so `rustc` can use its linker.  The simplest way is to get the
 [Visual Studio Build Tools] and check the “C++ build tools” workload.
 
+Windows 10 SDK is also required now. 
```

Comment:
We would certainly want to phrase this in the present, that is, not "now".

Reply:
The link is also changed. There isn't a c++ build tools.